Taarafo.Core/Brokers/Storages/StorageBroker.PostReports.cs

```diff
@@ -13,7 +14,10 @@ public partial class StorageBroker
     {
         public DbSet<PostReport> PostReports { get; set; }
 
+        public async ValueTask<PostReport> InsertPostReportAsync(PostReport postReport) =>
```

Comment:
This PR is for Delete Group Memebership.  This is out of scope for this change please exclude this change and do this as a seperate PR.